python/triton/common/build.py

```diff
@@ -27,6 +27,9 @@ def libcuda_dirs():
     # libcuda.so.1 (libc6,x86-64) => /lib/x86_64-linux-gnu/libcuda.so.1
     locs = [line.split()[-1] for line in libs.splitlines() if "libcuda.so" in line]
     dirs = [os.path.dirname(loc) for loc in locs]
+    env_ld_library_path = os.getenv("LD_LIBRARY_PATH")
+    if env_ld_library_path and not dirs:
```

Comment:
I tried to apply this patch but realized that it didn't solve my problem.

Without this patch, I got this error:

```
AssertionError: libcuda.so cannot found!
Possible files are located at ['/lib64/libcuda.so.1'].Please create a symlink of libcuda.so to any of the file.
```

But after setting LD_LIBRARY_PATH the error persists.

This is because "dirs" is not empty (it has the parent folder of "/lib64/libcuda.so.1"), but triton was looking for a file named "libcuda.so".